docs/how_to_build_images.md

```diff
@@ -0,0 +1,36 @@
+# How to build images
+
+Use appropriate playbook inside `playbooks` directory to build the images you need.
+
+To build localhost/foreman:2.1-42 image:
+
+```
+ansible-playbook playbooks/build-foreman-image.yml -e version=2.1 -e version_build=42
+```
+
+To build awesome-repo/foreman:2.1-1 and push into the registry:
+
+```
+ansible-playbook playbooks/build-foreman-image.yml -e version=2.1 -e registry=awesome-repo -e registry_user=admin -e registry_token=changeme -e push=true
+```
+
+It is possible to use additional snippets to modify an image build which is useful when you need to copy additional files into an image:
```

Comment:
The trickier part of this is - how does a user maintain and track this if they are pulling new updates to this git repository ? Can a user keep a `user_snippets` directory that is in `.gitignore`  and passes paths to those files to the playbook?

Reply:
That is a tricky one. I did not find a way how to configure an additional path for jinja templates/snippets outside of the role.  

I copy the files into appropriate places and remove them afterwards which is not great but does the job.